src/Field.ts

```diff
@@ -12,6 +19,8 @@ import {
   parseInput,
   defaultClearValue } from './parser';
 
+const mobxAny = mobx as any;
```

Comment:
To get the asMap function from Mobx3 we need to have an effectively untyped version of Mobx. We do that here.